fetch-quotes.js

```diff
@@ -0,0 +1,21 @@
+const request = require('superagent');
```

Comment:
not a comment on the code, but love how organized your pull requests were for this, made it easy to read and nice commit history.  Great job, Katie!